js/bootstrap-slider.js

```diff
@@ -231,7 +231,17 @@
 				toValue: function(percentage) {
 					var min = (this.options.min === 0) ? 0 : Math.log(this.options.min);
 					var max = Math.log(this.options.max);
-					return Math.exp(min + (max - min) * percentage / 100);
+					var value = Math.exp(min + (max - min) * percentage / 100);
+					value = this.options.min + Math.round((value - this.options.min) / this.options.step) * this.options.step;
```

Comment:
@rovolution Line 234 calculates the unstepped value.  Line 235 uses the value calculated in line 234 and rounds it to a stepped value.  It seemed clearer to have it in two lines than in one.